Task_TheFlexGame/index.css

```diff
@@ -1 +1,85 @@
-/* Start coding below for FLEX GAME TASK*/
+.container{
+    width: 2828px;
+    height: 2562px;
+    display: flex;
+    justify-content: space-between;
+}
+.menu{
+    width: 639px;
+    height: 2562px;
+    background: #604AC6;
+    display: flex;
+    flex-direction: column;
+    padding: 84.5px;
+    box-sizing: border-box;
+}
+.content{
+    width: 2145px;
+    height: 2562px;
+    background: #01AB63;
+    padding: 84.5px;
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+}
+.menu-title{
+    width: 590px;
+    height: 120px;

```

Comment:
Consider specifying a fallback font-family in case 'Inter' is not available on user's system.